### DISCUSSION TITLE
How can I change the order of attributes on <script>?

### DISCUSSION THREAD

**Discussion by jasonhibbs** [nuxtrdev/nuxtr-vscode#173] [Questions]
A small thing, but `<script setup lang="ts">` is seen throughout docs because "script setup" is the convention, so it’s weird that new components made with Nuxtr default to `<script lang="ts" setup>`.

**Comment by adhamfarrag** [comment]
Hello @jasonhibbs, 

Just got time to respond. Next release will include this change to follow the template as Vue, but in the time being you can create a [custom template](https://github.com/nuxtrdev/nuxtr-vscode?tab=readme-ov-file#personalized-vue-file-templates).
